### PR TITLE
Update dask-scheduler cli help text for preload

### PR DESCRIPTION
**Pull request by nicolls1**
As per discussion on #2111

### CODE REVIEW

File: distributed/cli/dask_scheduler.py [hunk 1]
```diff
@@ -59,8 +59,9 @@
 @click.option('--local-directory', default='', type=str,
               help="Directory to place scheduler files")
 @click.option('--preload', type=str, multiple=True, is_eager=True,
-              help='Module that should be loaded by each worker process '
-                   'like "foo.bar" or "/path/to/foo.py"')
+              help='Module that should be loaded by the scheduler process  '
+                   'like "foo.bar" or "/path/to/foo.py". Scheduler plugins '
+                   'should be loaded here.')
```
Comment (mrocklin) [inline distributed/cli/dask_scheduler.py]: Thanks for changing worker -> scheduler here. 

I'm not convinced that we should call out the specific use case of scheduler plugins though.  Preload scripts are used for a variety of reasons, of which this is only one.

Comment (mrocklin) [reply distributed/cli/dask_scheduler.py]: Should we remove this?

**Comment by mrocklin** [comment]
Merging.  Thanks @nicolls1 !

File: distributed/cli/dask_scheduler.py (hunk 1 above)
Comment (nicolls1) [reply distributed/cli/dask_scheduler.py]: Yeah good point, removed.

**Comment by mrocklin** [comment]
Also I notice that this is your first contribution to the project.  Welcome!

**Pull request merged**